pod/locale/fr/LC_MESSAGES/django.po

```diff
@@ -2132,6 +2138,98 @@ msgstr "Ils sont visibles sur la barre de lecture de la vidéo."
 msgid "You cannot enrich this video."
 msgstr "Vous ne pouvez pas enrichir cette vidéo."
 
+#: pod/favorite/models.py
+#: pod/favorite/templates/favorite/favorite_video_sort_select.html
+#: pod/recorder/models.py pod/video/models.py
+#: pod/video/templates/videos/video_sort_select.html
+msgid "Date added"
+msgstr "Date d’ajout"
+
+#: pod/favorite/models.py
+#: pod/favorite/templates/favorite/favorite_video_sort_select.html
+msgid "Rank"
+msgstr "Rang"
+
+#: pod/favorite/models.py
+msgid "Favorite"
+msgstr "Favori"
+
+#: pod/favorite/models.py
+msgid "Favorites"
+msgstr "Favoris"
+
+#: pod/favorite/templates/favorite/favorite_video_list.html
+#: pod/video/templates/videos/video_list.html
+msgid "Sorry, no video found."
+msgstr "Désolé, aucune vidéo trouvée."
+
+#: pod/favorite/templates/favorite/favorite_video_sort_select.html
+#: pod/video/templates/videos/video_sort_select.html
+msgid "Sort"
+msgstr "Tri"
+
+#: pod/favorite/templates/favorite/favorite_video_sort_select.html
+#: pod/video/templates/videos/video_sort_select.html
+msgid "Descending sort"
+msgstr "Tri descendant"
+
+#: pod/favorite/templates/favorite/favorite_video_sort_select.html
+#: pod/video/templates/videos/video_sort_select.html
+msgid "Sort Direction"
+msgstr "Direction de tri"
+
+#: pod/favorite/templates/favorite/favorite_video_sort_select.html
+#: pod/meeting/forms.py pod/meeting/templates/meeting/recordings.html
+#: pod/playlist/templates/playlist/playlist_element_list.html
+#: pod/video/feeds.py pod/video/models.py
+#: pod/video/templates/videos/video_sort_select.html
+msgid "Duration"
+msgstr "Durée"
+
+#: pod/favorite/templates/favorite/favorite_videos.html
+#: pod/video/templates/channel/channel.html
+#: pod/video/templates/videos/my_videos.html
+#: pod/video/templates/videos/videos.html
+#: pod/video_search/templates/search/search.html
+#, python-format
+msgid "%(counter)s video found"
+msgid_plural "%(counter)s videos found"
+msgstr[0] "%(counter)s vidéo trouvée"
+msgstr[1] "%(counter)s vidéos trouvées"
+
+#: pod/favorite/templates/favorite/favorite_videos.html
+#: pod/favorite/tests/test_views.py pod/main/templates/navbar.html
+msgid "My favorite videos"
+msgstr "Mes vidéos favorites"
+
+#: pod/favorite/templates/favorite/favorite_videos.html
+msgid "Reorganize your favorite videos"
+msgstr "Réorganiser vos vidéos favorites"
+
+#: pod/favorite/templates/favorite/favorite_videos.html
+msgid "Reorganize"
+msgstr "Réorganiser"
+
+#: pod/favorite/templates/favorite/favorite_videos.html
+msgid "Help - Drag & Drop"
+msgstr "Aide - Glisser & Déposer"
+
+#: pod/favorite/templates/favorite/favorite_videos.html
+msgid "Select the video you want by clicking and holding the mouse button."
+msgstr "Sélectionnez la vidéo que vous souhaitez en cliquant et maintenant le bouton de la souris."
+
+#: pod/favorite/templates/favorite/favorite_videos.html
+msgid "While holding down the mouse button, drag it to the desired location."
```

Comment:
idem, pour rester agnostique du terminal employé (souris ou tactile), je remplacerais le "the mouse button" par the video